Documentation/ignition.md

```diff
@@ -23,38 +23,13 @@ Fuze template files can be added in the `/var/lib/matchbox/ignition` directory o
  └── profiles
 ```
 
-### Reference
+## Referencing in Profiles
 
-Reference an Fuze config in a [Profile](matchbox.md#profiles) with `ignition_id`. When PXE booting, use the kernel option `coreos.first_boot=1` and `coreos.config.url` to point to the `matchbox` [Ignition endpoint](api.md#ignition-config).
-
-### Migration from v0.3.0
-
-In v0.4.0, `matchbox` switched to using the CoreOS [fuze](https://github.com/coreos/fuze) library, which formalizes and improves upon the YAML to Ignition JSON transform. Fuze provides better support for Ignition 2.0.0+, handles file content encoding, patches Ignition bugs, and performs better validations.
-
-Upgrade your Ignition YAML templates to match the [Fuze config schema](https://github.com/coreos/fuze/blob/master/doc/configuration.md). Typically, you'll need to do the following:
-
-* Remove `ignition_version: 1`, Fuze configs are version-less
-* Update `filesystems` section and set the `name`
-* Update `files` section to use `inline` as shown below
-* Replace `uid` and `gid` with `user` and `group` objects as shown above
-
-Maintain readable inline file contents in Fuze:
-
-```
-...
-files:
-  - path: /etc/foo.conf
-    filesystem: root
-    contents:
-      inline: |
-        foo bar
-```
-
-Support for the older Ignition v1 format has been dropped, so CoreOS machines must be **1010.1.0 or newer**. Read the upstream Ignition v1 to 2.0.0 [migration guide](https://coreos.com/ignition/docs/latest/migrating-configs.html) to understand the reasons behind schema changes.
+Profiles can include a Container Linux Config for provisioning machines. Specify the Container Linux Config in a [Profile](matchbox.md#profiles) with `ignition_id`. When PXE booting, use the kernel option `coreos.first_boot=1` and `coreos.config.url` to point to the `matchbox` [Ignition endpoint](api.md#ignition-config).
 
 ## Examples
 
-Here is an example Fuze template. This template will be rendered into a Fuze config (YAML), using group metadata, selectors, and query params as template variables. Finally, the Fuze config is served to client machines as Ignition JSON.
+Here is an example Container Linux Config template. Variables will be interpreted using group metadata, selectors, and query params. Matchbox will convert the config to Ignition to serve Container Linux machines.
```

Comment:
I think Github is playing tricks. It should be a Container Linux Config followed by the Ignition config that would be served. If you click expand in Github, it expands to show the Ignition first.

Full View https://github.com/coreos/matchbox/blob/28c6da21b92d25bc49719bc2b034d6b07acf72db/Documentation/ignition.md